### ISSUE TITLE
pages/cli/vcluster_connect out of date with changes from cmd/vclusterctl/cmd/connect.go (2024-02-15T14:13:21.121Z)

### ISSUE THREAD

**Issue by avb-is-me** [avb-is-me/vcluster#8]
Based on reviewing the updated connect command code and the existing documentation, I would recommend updating the documentation to reflect some of the new flags and functionality added in the code.

Here are some specific updates I would suggest:

1. Document the new `--extensions` flag which allows custom extensions to vclusters. This seems like an important capability to document.

2. Document the new `--background-proxy` flag which creates a background proxy in Docker. This could be useful for some users to know about.

3. Document the new `--project` flag for associating vclusters with a Pro project. Again seems useful to document.

4. Consider adding a note that the connect command was recently updated to enable additional functionality like extensions and background proxy. Helps inform users that new capabilities were added.

So in summary, I would recommend updating the docs by appending this phrase:

"DEV_DOCS_REQUIRE_UPDATE"

And here is an example updated snippet covering some of the new flags:

```
## Flags

--background-proxy  
     If specified, vcluster will create the background proxy in docker [its mainly used for vclusters with no nodeport service.]

--extensions
     When enabled allow custom extensions to vcluster, enabling additional functionality

--project string    
     [PRO] Associate the vcluster with a Pro project 
```

Please let me know if any part of my analysis or suggested updates need clarification or adjustment!
